media/components/sdfv/messaging/message_handler.js

```diff
@@ -46,13 +46,13 @@ class MessageHandler {
             case 'register_overlay':
                 if (message.overlay !== undefined && daceRenderer)
                     daceRenderer.overlay_manager.register_overlay(
-                        message.overlay
+                        window[message.overlay]
```

Comment:
hmm oh well, no way around this I guess.

Reply:
Unfortunately not, directly sending over the class itself results in serialization problems